pages/common/hyperfine.md

```diff
@@ -0,0 +1,23 @@
+# hyperfine
+
+> A command-line benchmarking tool.
+
+- Run a basic benchmark:
+
+`hyperfine '{{make}}'`
+
+- Change minimum number of benchmarking runs:
```

Comment:
What is the default number

Reply:
In the docs it says
>By default, it will perform _at least_ 10 benchmarking runs.

Which is... a bit confusing.

Reply:
Got it. It keeps on running the benchmarks until it has achieved a minimum variance to decide that the benchmark is stable. I just wanted the no. 10. We should clarify that.

Reply:
Oh ok 👍